pyoverkiz/client.py

```diff
@@ -809,7 +809,7 @@ async def check_response(response: ClientResponse) -> None:
             ) from error
 
         if result.get("errorCode"):
-            message = result.get("error").strip("'")
+            message = result.get("error").strip('"')
```

Comment:
If we don’t add any test, it would be nice to add as comment an example of why we trip the ".

Reply:
Agree! Preferably both 😄.